mycroft/listener/mic.py

```diff
@@ -47,6 +47,7 @@
 )
 from mycroft.util.log import LOG
 from ovos_utils.configuration import get_xdg_data_save_path
+from mycroft.util.audio_utils import play_audio_file
```

Comment:
per discussion, this should import from `ovos_utils.sound`

Reply:
lets do this in another PR, the util in core itself should be migrated

the signature is a bit different and it does not have ogg123 support, so let's clean that up before the move